projects/python-client/src/datapane/client/api/report/core.py

```diff
@@ -588,15 +598,14 @@ def serve(
             formatting: Sets the basic report styling; note that this is ignored if a report exists at the specified path
         """
         if not osp.isdir(path):
-            self.build(path, formatting=formatting)
+            self.build(path, formatting=formatting, compress_assets=True)
 
         os.chdir(path)  # Run the server in the specified path
-        server = HTTPServer((host, port), DPServer)
+        server = HTTPServer((host, port), CompressedAssetsHTTPHandler)
```

Comment:
do we need to pass `compress_assets` into the HTTPHandler here so it knows whether to set the content encoding header accordingly or not?

Reply:
Following https://github.com/datapane/datapane/pull/275#discussion_r965277222 `Report.serve` is set to always compress assets. I can add a `compress_assets` flag to `Report.serve` too if that would be handy

Reply:
Oh right - that's correct - ok let's leave for now then.